src/Sarif.Viewer.VisualStudio/String.extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Sarif.Viewer
+{
+    static class StringExtensions
```

Comment:
>StringExtensions [](start = 17, length = 16)

Do me a favor, zap this class, actually, and go add a static helper to TelemetryEvent or TelemetryProvider that does this construction.

problems here (all small)
1) new type with a single method
2) extension method that applies a method to a commonly used type where the method is only applicable in a small number of scenarios where that type is used

alternately, we could have put this extension method in a sub-namespace that would restrict use in contexts where we opt into the telemetry. that would entail putting the telemetry types in their own namespace.

Reply:
I'd rather just yank it and do KVP construction directly.